src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2010.ProvideCodeSnippets_WithArtifacts_Valid.sarif

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "CodeScanner",
+          "version": "1.0"
+        }
+      },
+      "versionControlProvenance": [
```

Comment:
>versionControlProvenance [](start = 7, length = 24)

FYI: with the changes I merged this morning, it is no longer necessary for every test file to obey every analysis rule -- because the test framework now only looks at results that come from the rule you are testing. So, for example, you don't need to supply `versionControlProvenance`.